src/routes/formsgSiteLaunch.ts

```diff
@@ -229,11 +272,26 @@ export class FormsgSiteLaunchRouter {
         formResponses.map(this.launchSiteFromForm)
       )
       const successResults: DnsRecordsEmailProps[] = []
-      launchResults.forEach((launchResult) => {
+      for (const launchResult of launchResults) {
         if (launchResult.isOk()) {
-          successResults.push(launchResult.value)
+          // check for AAAA records
+          const digReponse = await this.digDomainForQuadARecords(
+            launchResult.value.primaryDomainSource,
+            "AAAA"
+          )
+          const successResult: DnsRecordsEmailProps = launchResult.value
+          if (digReponse.answer) {
```

Comment:
good point, since this is a non-core flow (to check quad A records), I didn't add the else. But I guess it will be good to log that anyways. Have add a logger.info

Reply:
@seaerchin I realised that the dig function was throwing err. But since this will land into the catch flow and dig for AAAA records is rather a "non-core" flow, instead of throwing I made it return null so that we can skip the quad-A check and proceed

Reply:
cc @kishore03109